package.json

```diff
@@ -8,9 +8,11 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "classnames": "^2.2.6",
```

Comment:
I am not using this yet, but figured might need it sooner or later so left it in here.